tools/generator/src/Extensions/Dictionary+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+extension Dictionary {
+    /// Look up a value using the key. Throw an error if it is not found.
```

Comment:
Can you add this in its own PR, along with changing code to use this new method?

Reply:
I have added the new method in #805. I am happy to help with switching code over to use this new method. However, I don't think that this should hold up this PR.